Update contributing

RedFox0x20 opened the pull request:
Add additional step regarding `eslint` to help guide new contributors on ensuring their changes follow the styling of the project.

audreyfeldroy commented on CONTRIBUTING.md:
```diff
@@ -19,7 +19,8 @@ Want to contribute to this repo? That's great! These instructions will help you
 2. Please create a new branch and name it with the changes you are making. E.g. `fix-card-sort-order`
 3. Make your code changes. You can preview your changes by [Running the project locally](#running-the-project-locally)
-4. Send us a pull request.
+4. Run `eslint . --fix` to ensure your code is following our linter rules, you may have to fix some issues manually.
```
```suggestion
4. Run `eslint . --fix` and `yarn format` to ensure your code is following our linter rules. You may have to fix some issues manually.
```

audreyfeldroy reviewed (changes requested):
Thanks for thinking to add this @RedFox0x20. It'll save a lot of reviewer time. I added `yarn format` separately, but perhaps we can soon combine those two into the same `yarn format` command.

RedFox0x20 commented:
> Thanks for thinking to add this @RedFox0x20. It'll save a lot of reviewer time. I added `yarn format` separately, but perhaps we can soon combine those two into the same `yarn format` command.

Yes, when I saw your changes I thought this too haha. Added your text changes to the commit too.

audreyfeldroy reviewed (approved):
Nice work combining the commands into `yarn format` @RedFox0x20. Merging this, thanks!
